packages/venia-ui/lib/components/MiniCart/miniCart.js

```diff
@@ -32,9 +32,13 @@ const MiniCart = React.forwardRef((props, ref) => {
     // when the MiniCart is open.
     useScrollLock(isOpen);
 
+    const { queries, mutations } = { ...MiniCartOperations };
     const talonProps = useMiniCart({
         setIsOpen,
-        ...MiniCartOperations
+        operations: {
```

Comment:
How about sending the `MiniCartOperations` directly here?

Reply:
I have changed exported format in the `packages/venia-ui/lib/components/MiniCart/miniCart.gql.js` so it is just `operations` now